includes/utils.php

```diff
@@ -285,14 +285,28 @@ function available_pull_post_types( $connection, $type ) {
 /**
  * Return post types that are allowed to be distributed
  *
+ * @param string $output Optional. The type of output to return.
+ *                       Accepts post type 'names' or 'objects'. Default 'names'.
+ *
  * @since  1.0
  * @return array
  */
-function distributable_post_types() {
-	$post_types = get_post_types( [ 'public' => true ] );
+function distributable_post_types( $output = 'names' ) {
+	$post_types = get_post_types( [ 'show_in_rest' => true ], $output );
+
+	$exclude_post_types = [
+		'attachment',
+		'nav_menu_item',
+		'wp_block',
+		'wp_navigation',
+		'wp_template',
+		'wp_template_part',
+		'dt_ext_connection',
+		'dt_subscription',
+	];
 
```

Comment:
```suggestion
	$exclude_post_types = [
		'attachment',
		'dt_subscription',
	];
```

This removes unneeded items inf you check for public post types per the comment above.